politeiawww/backend.go

```diff
@@ -967,6 +997,38 @@ func (b *backend) ProcessProposalDetails(propDetails www.ProposalsDetails, isUse
 	return &reply, nil
 }
 
+// ProcessComment processes a submitted
```

Comment:
I think you forgot to complete this comment. :)

Reply:
Done